pacmacs-image.el

```diff
@@ -91,53 +120,117 @@
+          (pacmacs--wall-bits-get-corners wall-bits)))
+    (when left-upper
+      (pacmacs--put-wall-tile-corner wall-tile 0 0 weight
+                                     (pacmacs--two-weights-to-color nil nil nil weight)))
+
+    (when right-upper
+      (pacmacs--put-wall-tile-corner wall-tile 0 (- width weight) weight
+                                     (pacmacs--two-weights-to-color nil t nil weight)))
+
+    (when left-bottom
+      (pacmacs--put-wall-tile-corner wall-tile (- height weight) 0 weight
+                                     (pacmacs--two-weights-to-color t nil nil weight)))
+
+    (when right-bottom
+      (pacmacs--put-wall-tile-corner wall-tile (- height weight) (- width weight) weight
+                                     (pacmacs--two-weights-to-color t t nil weight)))))
+
+(defun pacmacs--put-bars (wall-tile width height weight wall-bits)
+  (-let (((bottom right top left)
+          (pacmacs--wall-bits-get-bars wall-bits)))
+    (when left
+      (pacmacs--put-vertical-bar wall-tile 0 height weight #'identity))
+
+    (when right
+      (pacmacs--put-vertical-bar wall-tile (- width weight) height weight
+                                 (pacmacs--inverted-weight-to-color weight)))
+
+    (when top
+      (pacmacs--put-horizontal-bar wall-tile 0 width weight #'identity))
+    
+    (when bottom
+      (pacmacs--put-horizontal-bar wall-tile (- height weight) width weight
+                                   (pacmacs--inverted-weight-to-color weight)))))
+
+(defun pacmacs--put-outer-corners (wall-tile width height weight wall-bits)
+  (-let (((bottom right top left)
+          (pacmacs--wall-bits-get-bars wall-bits)))
+    (when (and left top) ;left-upper
+      (pacmacs--put-wall-tile-corner wall-tile 0 0 weight
+                                     (pacmacs--two-weights-to-color t t t weight)))
+
+    (when (and right top) ;right-upper
+      (pacmacs--put-wall-tile-corner wall-tile 0 (- width weight) weight
+                                     (pacmacs--two-weights-to-color t nil t weight)))
+
+    (when (and left bottom) ;left-bottom
+      (pacmacs--put-wall-tile-corner wall-tile (- height weight) 0 weight
+                                     (pacmacs--two-weights-to-color nil t t weight)))
+
+    (when (and right bottom) ;right-bottom
+      (pacmacs--put-wall-tile-corner wall-tile (- height weight) (- width weight) weight
+                                     (pacmacs--two-weights-to-color nil nil t weight)))))
+
 (defun pacmacs--create-wall-tile (width height
-                                  bottom right
-                                  top left
-                                  left-upper right-upper
-                                  left-bottom right-bottom)
+                                        bottom right
+                                        top left
+                                        left-upper right-upper
+                                        left-bottom right-bottom)
```

Comment:
It's not me! It's Emacs! Its identation thinks that the first element of the list is a function name and everything is aligned as the function's arguments.

It's not a big deal, actually, but I'll fix it anyway. Thanks!